tools_webrtc/ios/build_ios_libs.py

```diff
@@ -59,7 +59,7 @@ def _ParseArgs():
            'If specified together with -c, deletes the dir.')
   parser.add_argument('-r', '--revision', type=int, default=0,
       help='Specifies a revision number to embed if building the framework.')
-  parser.add_argument('-e', '--bitcode', action='store_true', default=True,
+  parser.add_argument('-e', '--bitcode', action='store_true', default=False,
```

Comment:
테스트를 위해서만 비트코드를 끄는 건가요?

Reply:
Default option is `False`, but it was changed in the main branch. Because we don't really need bitcode for debugging or testing, I've turned off the default value.